models/baseModels/SalesQuote/SalesQuote.ts

```diff
@@ -48,6 +51,14 @@ export class SalesQuote extends Invoice {
     return invoice;
   }
 
+  static filters: FiltersMap = {};
+
+  async afterSubmit(): Promise<void> {
+    await super.afterSubmit();
+    const leadData = await this.fyo.doc.getDoc(ModelNameEnum.Lead, this.party);
+    await leadData.setAndSync('status', 'Quotation');
+  }
```

Comment:
These lines of code should only be performed if `SalesQuote` is created from `Lead`

You can get the linked leadDoc like below,
```ts
await this.loadAndGetLink('party')
```